src/modules/cmdpal/Microsoft.CmdPal.UI/Converters/FilterTemplateSelector.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Core.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.CmdPal.UI;
+
+internal sealed partial class FilterTemplateSelector : DataTemplateSelector
+{
+    public DataTemplate? Default { get; set; }
+
+    public DataTemplate? Separator { get; set; }
+
+    protected override DataTemplate? SelectTemplateCore(object item, DependencyObject dependencyObject)
+    {
+        DataTemplate? dataTemplate = Default;
+
+        if (dependencyObject is ComboBoxItem comboBoxItem)
+        {
+            comboBoxItem.IsEnabled = true;
+
+            if (item is SeparatorViewModel)
+            {
+                comboBoxItem.IsEnabled = false;
+                comboBoxItem.AllowFocusWhenDisabled = false;
+                comboBoxItem.AllowFocusOnInteraction = false;
```

Comment:
do we need to set these other properties to `true` if it's _not_ a separator?

Reply:
I _believe_ these default to true. The `AllowFocusWhenDisabled` only applies if the element is disabled.